src/ranged.cpp

```diff
@@ -1862,13 +1862,16 @@ item::sound_data item::gun_noise( bool const burst ) const
 
     noise = std::max( noise, 0 );
 
+    // Grenade launchers
     if( ammo_type() == ammotype( "40mm" ) ) {
         return { 8, _( "Thunk!" ) };
 
-    } else if( typeId() == "hk_g80") {
+    // Railguns
+    } else if( ammotype() == "12mm" || ammotype( "metal_rail" ) ) {
```

Comment:
Oh man, thank you. Going back through this and flamethrowers weren't producing the right sound effect anymore. *d'oh*